js/books.js

```diff
@@ -0,0 +1,63 @@
+// eslint-disable-next-line import/extensions
+import Storage from './localStorage.js';
+// eslint-disable-next-line import/extensions
```

Comment:
- [ ] Let's try to not disable linters as they help us write cleaner code.

Reply:
The problem here @Ghiftee is that eslint will ask me to remove file extensions and that will result int an error because removing the file extensions means that the imports will not work. That is why I disabled it at that point.
![Screenshot from 2023-01-13 00-01-12](https://user-images.githubusercontent.com/55017675/212180379-107bfdd2-1329-421a-ab45-f2732ef28b2e.png)